src/Components/HomePage/Styles/FlipCard.css

```diff
@@ -0,0 +1,54 @@
+.CarouselCard{
+    height:740px;
```

Comment:
As we discussed in the meeting, explore the unit `vw` and `vh` for auto adjustment on size